lib/sycamore/sycamore/tests/manual/test_fast_sycamore_import.py

```diff
@@ -17,21 +17,21 @@ def test_disallow_unconditional_dependencies():
     import sycamore
 
     _ = sycamore  # make sure tools don't remove the import
-    for m in ["guidance", "neo4j", "ray", "torch"]:
+    for m in ["guidance", "neo4j", "ray", "torch", "duckdb", "elasticsearch", "opensearchpy", "pinecone", "weaviate"]:
```

Comment:
can you switch this to a constant so we don't end up repeating it three times?  I probably should have done that before.

Reply:
Oh, yeah. Definitely. I thought about this, and honestly thought I had done it, but now I can't find any evidence of it. Guess I'm hallucinating now too.